jira/client.py

```diff
@@ -4516,13 +4516,27 @@ def sprints(
             self.AGILE_BASE_URL,
         )
 
-    def sprints_by_name(self, id, extended=False):
+    def sprints_by_name(self, id, extended=False, state=None):
+        """Get a dictionary of sprint Resources where the name of the sprint is the key.
+
+        Args:
+            board_id (int): the board to get sprints from
+            extended (bool): Deprecated.
+            state (str): Filters results to sprints in specified states. Valid values: `future`, `active`, `closed`.
+              You can define multiple states separated by commas
```

Comment:
is this case sensitive?

Reply:
no clue didnt validate this just copied from sprints()

Reply:
beyond this PR then